code/modules/client/client_procs.dm

```diff
@@ -312,7 +312,9 @@ GLOBAL_LIST_INIT(blacklisted_builds, list(
 			else
 				qdel(src)
 				return
-
+	if(byond_build > MAX_RECOMMENDED_CLIENT)
+		to_chat(src, "<span class='userdanger'>Your version of byond is over the maximum recommended build number for clients ([MAX_RECOMMENDED_CLIENT]) and is likely to be very buggy.</span>")
```

Comment:
```suggestion
		to_chat(src, "<span class='userdanger'>Your version of byond is over the maximum recommended build version for clients ([MAX_RECOMMENDED_CLIENT]) and may be unstable.</span>")
```

Reply:
"build number" is used more often and sounds more natural to me.